ui/components/MesherySnackbarWrapper.js

```diff
@@ -62,6 +62,9 @@ const styles = (theme) => ({
     alignItems : "center",
   },
   snackbarContent : { [theme.breakpoints.up("sm")] : { minWidth : "344px !important", }, },
+  snackbarContentBorder : {
+    border : "1px solid rgba(102, 102, 102, 1)"
```

Comment:
This color isn't available in the theme?

Reply:
Not available in the theme, But It is used as `iconColor` at line number 50 in the same file.